posts/2021/08/github-actions-benchmarks.md

```diff
@@ -0,0 +1,336 @@
+<!--
+.. title: Is GitHub Actions suitable for running benchmarks?
+.. slug: github-actions-benchmarks
+.. date: 2021-08-30 00:01 UTC
+.. tags: github-actions, Open-Source, continuous-integration, performance
+.. category:
+.. link:
+.. description:
+.. type: markdown
+.. author: Jaime Rodríguez-Guerra
+.. previewimage: /images/2021/08/github-actions-benchmark.png
+.. image:: /images/2021/08/github-actions-benchmark.png
+   :alt: Reliability of benchmarks in GitHub Actions
+-->
+
+![Reliability of benchmarks in GitHub Actions](/images/2021/08/github-actions-benchmark.png)
```

Comment:
It is a bit ambiguous at this point (before reading the below text) what the individual dots here represent, but maybe it is still best to keep this colorful figure up front to grab attention?

Reply:
Yes, I agree. I only added it here for eye candy, but if needed we can think of a different banner?